source/agora/common/Config.d

```diff
@@ -1068,7 +1065,10 @@ noexist_event_handlers:
 `;
         auto node = Loader.fromString(conf_example).load();
         auto handlers = parserEventHandlers("event_handlers" in node, cmdln);
-        assert(handlers.length == 0);
+        assert(handlers[0].handler_addresses.length == 0);
+        assert(handlers[1].handler_addresses.length == 0);
+        assert(handlers[2].handler_addresses.length == 0);
+        assert(handlers[3].handler_addresses.length == 0);
```

Comment:
`foreach` + keep the original `assert` ?

Reply:
Updated